docs/src/tutorials/yocto-linux.md

```diff
@@ -0,0 +1,102 @@
+
+> Most of the installation process is based on [Yocto Project Quick Build
+guide](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html).
+
+Before starting, be sure your hosts meets the following requirements:
+
+- 50 Gbytes of free disk space
+of Linux distributions that support the Yocto Project, see the Supported Linux Distributions section in the Yocto
+Project Reference Manual. For detailed information on preparing your build host, see the Preparing the Build Host
+section in the Yocto Project Development Tasks Manual.
+- Git 1.8.3.1 or greater
+- tar 1.28 or greater
+- Python 3.6.0 or greater.
+- gcc 5.0 or greater.
+
+For the purposes of the tutorial, we assume `~/yocto-thinedge` working directory. If using other directory, be sure to
+change paths where needed.
+
+### Build Host Packages
+
+Install essential packages:
+
+```bash
+$ sudo apt install gawk wget git diffstat unzip texinfo gcc build-essential chrpath socat cpio python3 python3-pip python3-pexpect xz-utils debianutils iputils-ping python3-git python3-jinja2 libegl1-mesa libsdl1.2-dev pylint3 xterm python3-subunit mesa-common-dev zstd liblz4-tool
+```
+
+### Clone Poky Linux repository
+
+Clone the Poky Linux distribution sources. We'll be using version `kirkstone`. You can use `--depth=1` to speed up the
+process. If using these options, to see previous commits or other branches see
+[here](https://stackoverflow.com/questions/29270058/how-to-fetch-all-git-history-after-i-clone-the-repo-with-depth-1)
+and [here](https://stackoverflow.com/questions/17714159/how-do-i-undo-a-single-branch-clone).
+
+```
+$ git clone git://git.yoctoproject.org/poky --branch=kirkstone --depth=1
+```
+
+### Add layers
+
+We'll be using `meta-tedge` and `meta-openembedded`. First, fetch the repositories:
+
+```bash
+git clone https://github.com/Bravo555/meta-tedge.git
+git clone git://git.openembedded.org/meta-openembedded
+```
+
+Next, add the following layers:
+
+```bash
+bitbake-layers add-layer ~/yocto-thinedge/meta-openembedded/meta-oe
+bitbake-layers add-layer ~/yocto-thinedge/meta-openembedded/meta-networking
+bitbake-layers add-layer ~/yocto-thinedge/meta-tedge
+
+```
+
+### Configure the build
+
+Activate `Systemd` as default init manager by adding following lines to `local.conf`:
+
+```
+DISTRO_FEATURES:append = " systemd"
+DISTRO_FEATURES_BACKFILL_CONSIDERED += "sysvinit"
+VIRTUAL-RUNTIME_init_manager = "systemd"
+VIRTUAL-RUNTIME_initscripts = "systemd-compat-units"
+```
+
+Build `tedge` by running following command :
+
+```bash
+$ bitbake tedge-image
+```
+
+### Run the build
+
+When the build it complete, run it with qemu:
+
+```bash
+$ runqemu nographic
+```
+
+## Further recommendations
+
+After building the reference distribution and image, you can explore creating your own layer and image, and then
+integrating `tedge-*` recipes for it. To make Yocto run on your hardware, check other layers in the [OpenEmbedded Layer
```

Comment:
For a tutorial, I would add here some details on how to run yocto+tedge on some hardware.

Reply:
@didier-wenzek  I added exemplary instruction for raspberry pi devices. To avoid repetition, I made it as continuation of previous segment, because it requires only one additional layer and some config changes.

Reply:
I have also changed some information that is no longer valid (we have one additional supported version and image name is different now)

Reply:
Seems good. This will have to be tested with @gligorisaev.